test/Feature/SeedConcretizeMalloc.c

```diff
@@ -0,0 +1,27 @@
+// RUN: %clang -emit-llvm -c -g %s -o %t.bc
```

Comment:
I would also structure this test as the other two.  This also seems a bit more complicated than it should be.  I would just allocate a very large value, and then check via an assert that you get it back.